[2024-09-30T14:56:34.018+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-30T14:56:34.172+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT.fact_run.fact_1 scheduled__2024-09-29T00:00:00+00:00 [queued]>
[2024-09-30T14:56:34.179+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT.fact_run.fact_1 scheduled__2024-09-29T00:00:00+00:00 [queued]>
[2024-09-30T14:56:34.180+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-30T14:56:34.188+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fact_run.fact_1> on 2024-09-29 00:00:00+00:00
[2024-09-30T14:56:34.194+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1104) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-30T14:56:34.197+0000] {standard_task_runner.py:72} INFO - Started process 1112 to run task
[2024-09-30T14:56:34.198+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ELT', 'fact_run.fact_1', 'scheduled__2024-09-29T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmp99_lqj04']
[2024-09-30T14:56:34.201+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask fact_run.fact_1
[2024-09-30T14:56:34.239+0000] {task_command.py:467} INFO - Running <TaskInstance: ELT.fact_run.fact_1 scheduled__2024-09-29T00:00:00+00:00 [running]> on host 51a7ea638889
[2024-09-30T14:56:34.288+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ELT' AIRFLOW_CTX_TASK_ID='fact_run.fact_1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-29T00:00:00+00:00'
[2024-09-30T14:56:34.289+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-30T14:56:34.315+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-30T14:56:34.325+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-30T14:56:34.326+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ELT, task_id=fact_run.fact_1, run_id=scheduled__2024-09-29T00:00:00+00:00, execution_date=20240929T000000, start_date=20240930T145634, end_date=20240930T145634
[2024-09-30T14:56:34.377+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-30T14:56:34.386+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-30T14:56:34.387+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
